client/src/app/appCtrl.js

```diff
@@ -132,8 +136,10 @@ angular.module('app').controller('appCtrl', ['$scope', '$http', '$timeout', 'app
             fieldValue = iteration;
           }
           // check time monotonicity
-          if (fieldValue <= tmpTime) {
-            handleError("Your time is not monotonic at row "+rowId+"! Graphs are incorrect.", "danger", false);
+          if (fieldValue <= tmpTime && data[rowId][resetFieldIdx] !== 1) {
```

Comment:
fixed problem with OPF files, where `reset` could indicate a new sequence with new/independent times - that caused "refreshes" in the graph.